lib/vanagon/project.rb

```diff
@@ -46,6 +106,9 @@ def initialize(name, platform)
       @requires = []
       @directories = []
       @settings = {}
+      # Environments are like Hashes but with specific constraints
+      # around their keys and values.
+      @environment = Vanagon::Environment.new
```

Comment:
I assume there are going to be two seperate environment objects during a build. One defined here in the project, and one defined in the platform. How are those two going to interact?